srml/staking/src/lib.rs

```diff
@@ -97,12 +97,12 @@ impl Default for ValidatorPrefs {
 }
 
 #[derive(PartialEq, Eq, Clone, Encode, Decode, RuntimeDebug)]
-pub enum StakingBalance<RingBalance, KtonBalance> {
-	Ring(RingBalance),
-	Kton(KtonBalance),
+pub enum StakingBalance<Ring, Kton> {
```

Comment:
RingBalance looks good to me, since it represent a balance value.

Reply:
`Ring` and `Kton` are `StakingBalance`, I think it's clear enough.